src/main/java/org/opentripplanner/index/IndexGraphQLSchema.java

```diff
@@ -1291,10 +1358,225 @@ public IndexGraphQLSchema(GraphIndex index) {
+                .build())
+            .build();
+
+        final GraphQLObjectType itineraryType = GraphQLObjectType.newObject()
+            .name("Itinerary")
+            .field(GraphQLFieldDefinition.newFieldDefinition()
+                .name("startTime")
+                .description("Time that the trip departs.")
+                .type(Scalars.GraphQLLong)
+                .dataFetcher(environment -> ((Itinerary)environment.getSource()).startTime.getTime().getTime())
+                .build())
+            .field(GraphQLFieldDefinition.newFieldDefinition()
+                .name("endTime")
+                .description("Time that the trip arrives.")
+                .type(Scalars.GraphQLLong)
+                .dataFetcher(environment -> ((Itinerary)environment.getSource()).endTime.getTime().getTime())
+                .build())
+            .field(GraphQLFieldDefinition.newFieldDefinition()
+                .name("duration")
+                .description("Duration of the trip on this itinerary, in seconds.")
+                .type(Scalars.GraphQLLong)
+                .dataFetcher(environment -> ((Itinerary)environment.getSource()).duration)
+                .build())
+            .field(GraphQLFieldDefinition.newFieldDefinition()
+                .name("waitingTime")
+                .description("How much time is spent waiting for transit to arrive, in seconds.")
+                .type(Scalars.GraphQLLong)
+                .dataFetcher(environment -> ((Itinerary)environment.getSource()).waitingTime)
+                .build())
+            .field(GraphQLFieldDefinition.newFieldDefinition()
+                .name("walkTime")
+                .description("How much time is spent walking, in seconds.")
+                .type(Scalars.GraphQLLong)
+                .dataFetcher(environment -> ((Itinerary)environment.getSource()).walkTime)
+                .build())
+            .field(GraphQLFieldDefinition.newFieldDefinition()
+                .name("walkDistance")
+                .description("How far the user has to walk, in meters.")
+                .type(Scalars.GraphQLFloat)
+                .dataFetcher(environment -> ((Itinerary)environment.getSource()).walkDistance)
+                .build())
+            .field(GraphQLFieldDefinition.newFieldDefinition()
+                .name("legs")
+                .description("A list of Legs. Each Leg is either a walking (cycling, car) portion of the trip, or a transit trip on a particular vehicle. So a trip where the use walks to the Q train, transfers to the 6, then walks to their destination, has four legs.")
+                .type(new GraphQLNonNull(new GraphQLList(legType)))
+                .dataFetcher(environment -> ((Itinerary)environment.getSource()).legs)
+                .build())
+             .build();
+
+        planType = GraphQLObjectType.newObject()
+            .name("Plan")
+            .withInterface(nodeInterface)
```

Comment:
We can't really promise to be able to fetch the same instance of the plan later, so this shouldn't be implementing node. (Or a reverse implementation should be added [here](https://github.com/HSLdevcom/OpenTripPlanner/pull/43/files#diff-6261df6804def1193d18e405a513515cR1075))